lib/listener.js

```diff
@@ -16,64 +16,65 @@ Listener.prototype.startGame = function () {
 };
 
 Listener.prototype.setListeners = function () {
-  this.firing(this.player, this.projectiles);
-  this.moveRight(this.player);
-  this.moveLeft(this.player);
-  this.toggleProjectileType(this.player);
+  this.firing();
+  this.moveRight();
+  this.moveLeft();
+  this.toggleProjectileType();
 };
 
-Listener.prototype.toggleProjectileType = function (player) {
+Listener.prototype.toggleProjectileType = function () {
   this.canvas.addEventListener('keydown', function (event) {
     if (event.keyCode === 32) {
-      if (player.projectileType === 'blue') {
-        player.projectileType = 'orange';
-      } else if (player.projectileType === 'orange') {
-        player.projectileType = 'blue';
+      if (this.player.projectileType === 'blue') {
+        this.player.projectileType = 'orange';
+      } else if (this.player.projectileType === 'orange') {
+        this.player.projectileType = 'blue';
       }
```

Comment:
Since it looks like you are always comparing this.player.projectileType, it seems like you could use a switch/case statement if you want.